Resources/ProtobufGen/ProtobufGen/ProtobufGen.csproj

```diff
@@ -2,7 +2,9 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <RollForward>LatestMajor</RollForward>
```

Comment:
Isn't this the default option?

Reply:
Nope. I believe the default is `LatestMinor`.

Reply:
https://docs.microsoft.com/en-us/dotnet/core/whats-new/dotnet-core-3-0#runtimesdk

So freakin' complicated, apparently `Minor` is the default, thanks 😁

Reply:
Ah yeah that does make more sense...